pkg/security/security.go

```diff
@@ -384,7 +384,7 @@ func (ac *AuthContext) Header(header string) []string {
 			return meta.Get(header)
 		}
 	} else if ac.Request != nil {
-		return ac.Request.Header[header]
+		return ac.Request.Header.Values(header)
```

Comment:
For "x-forwarded-client-cert", i think it is no diff

Reply:
HTTP headers are not case sensitive, comparing literal values is a bug